doc/content/verification/ver-1e.md

```diff
@@ -0,0 +1,69 @@
+# val-1e
+
+# Diffusion in a composite layer
+
+# Depleting Source Problem
```

Comment:
```suggestion
# ver-1e

# Diffusion in Composite Material Layers
```

@singhgp4321 assuming that the depleting source title snuck in from a copy-paste. Feel free to disregard if both headings are necessary.

Reply:
@cticenhour Yes, a copy-paste error; just removed it.